merge-release-run.js

```diff
@@ -50,7 +50,10 @@ const run = async () => {
   process.stdout.write(execSync(`npm version --allow-same-version=true --git-tag-version=false ${current} `))
   let newVersion = execSync(`npm version --git-tag-version=false ${version}`).toString()
   console.log(newVersion)
+  process.stdout.write(execSync(`git commit -a --amend --no-edit`))
```

Comment:
Amending the commit because it seems like don't want to add the version bump commit.

Reply:
@mikeal just to make sure is this the behavior you want?

To be honest I would expect there to be a new commit for the version bump instead of changing the commit pushed

Reply:
So, I didn’t like this at first, because it puts a lot of noise into the commit log when you’re pushing directly into master. However, it’s actually quite nice when you’re doing “the right thing :)” and doing everything in branches/pr’s and there are often multiple commits in each release, so I’m fine with this change.